### ISSUE TITLE
Not working if I bind too many ports in docker-stack.yml ?

### ISSUE THREAD

**Issue by maple3142** [sr229/code-server-openshift#7]
I tried to start a simple node server in code-server, but it require additional ports to work. If I only bind `8888` and `9000` port like [this](https://gist.githubusercontent.com/maple3142/4bb3bc8570d852c66d072c856d1e8b3c/raw/b4d90aac456b57164773013f93765a1c96363b0e/docker-stack.yml), it will work in play with docker.

But the problem appears when I bind `8000-9000` ports like [this](https://gist.githubusercontent.com/maple3142/4bb3bc8570d852c66d072c856d1e8b3c/raw/de254764f3dff5292d0b0bdc55fefa960b31bc65/docker-stack.yml):
![image](https://user-images.githubusercontent.com/9370547/68214875-331de100-0019-11ea-8cda-9e94ab7e8411.png)
When I try to access port 9000, it just returns `ERR_EMPTY_RESPONSE` error.

**Comment by sr229** [comment]
Must be how v2 has changed something, I'll check it out later

**Comment by sr229** [comment]
Yeah I haven't reflected the new changes I made yet, will push tomorrow.
